src/main/resources/application.yaml

```diff
@@ -141,6 +141,18 @@ server:
     key-store-password: "password"
   port: 8443
 
+spring:
```

Comment:
adding data base connection in bulk processor is against the basic architecture.
this implementation doesn't respect multi-tenancy either